src/styles/global.css

```diff
@@ -1,10 +1,28 @@
 :root {
   --primary-color: #3f9ac9;
   --secondary-color: #e7e9ea;
+  --dark-grey-color: #666869;
 }
-body {
+body,
+html {
   color: var(--primary-color);
-  background-color: var(--secondary-color);
+  background-color: white;
```

Comment:
`background-color` property is assigned to a hard-coded color and is duplicated!